docs/_coverpage.md

```diff
@@ -0,0 +1,8 @@
+![logo](media/kubelinter-horizontal.png)
```

Comment:
Can't comment on the image, but can we use the `.svg` files instead of the `.png` files? (They are already checked in to the repo)